main.css

```diff
@@ -96,10 +98,12 @@ textarea {
 }
 
 .web {
-    float: left;
     background-color: white;
-    padding: 21px;
     height: 350px;
+    float: right;
```

Comment:
I think float right will not behave correctly because other two divs in the same element are float left. Can you test it on mobile device view and confirm?